### PR TITLE
Fixing modules and view engine

### PR DESCRIPTION
**Pull request by mboudreau**
A few improvements to the codebase, fixed an issue with JWT decoding.

### CODE REVIEW

**Review by rachellerathbone — APPROVED** [review]

**Review by rachellerathbone — APPROVED** [review]

**Comment by rachellerathbone** [comment]
Just checked out this branch and can't get the app to load. Was it working for you? I'm seeing

Screenshot 2023-03-17 at 12 07 14 pm

**Comment by krazziekay** [comment]
> Just checked out this branch and can't get the app to load. Was it working for you? I'm seeing
> 
> 
> 
> Screenshot 2023-03-17 at 12 07 14 pm
> 
> 
> 
> 

That happens if you've uninstalled the app from Jira when the app is not running. The JiraTenant wont be removed from the DB file and then next time when you run it it will still be taking the previous secretKey.
Remove the .db file and reinstalling will fix this.

File: src/views/layout.squirrelly [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>{{ it.title }}</title>
```
Comment (rachellerathbone) [inline src/views/layout.squirrelly:4]: Missing the * so any all titles will be `undefined`

**Review by rachellerathbone — APPROVED** [review]

File: src/views/layout.squirrelly (hunk 1 above)
Comment (mboudreau) [reply src/views/layout.squirrelly:4]: * is to unescape in squirrelly.  It's not needed for the title and seems to work fine on my local?

**Review by rachellerathbone — APPROVED** [review]

**Pull request merged**